packages/android_intent_plus/android/src/main/java/dev/fluttercommunity/plus/androidintent/IntentSender.java

```diff
@@ -172,10 +175,6 @@ Intent buildIntent(
       if (componentName != null) {
         intent.setComponent(componentName);
       }
-      if (intent.resolveActivity(applicationContext.getPackageManager()) == null) {
-        Log.i(TAG, "Cannot resolve explicit intent - ignoring package");
-        intent.setPackage(null);
```

Comment:
I think that just removing this line would be better. Let's leave the `Log.i`

Reply:
Sure I will do all the suggested changes and commit Thanks